polar-core/src/kb.rs

```diff
@@ -272,6 +302,12 @@ impl KnowledgeBase {
 \tPerhaps you meant to add an actor block to the top of your policy, like this:
 
 \t  actor {} {{}}", rule_instance.tag));
+                            } else if rule_type_instance.tag == sym!("Resource") {
```

Comment:
Something like this should work, with `RESOURCE_UNION_NAME` imported from `resource_block.rs`:

```suggestion
                            } else if rule_type_instance.tag.0 == RESOURCE_UNION_NAME {
```

Reply:
👍🏻 updated this and the other actor check on the previous line